Source/WebGPU/WebGPU/ShaderModule.mm

```diff
@@ -80,6 +80,36 @@
     return library;
 }
 
+// FIXME: https://bugs.webkit.org/show_bug.cgi?id=247030 - (Remove placeholder test shader)
+static String fallbackMSLString()
+{
+    return String::fromUTF8(
+        "#include <metal_stdlib>        \n"
+        "using namespace metal;         \n"
+        "struct Vertex {                \n"
+        "   float4 position [[position]];  \n"
+        "   float4 color;                  \n"
```

Comment:
Use C++ raw string literal?

```
return String::fromUTF8(R"(
#include <metal_stdlib>
using namespace metal;
struct Vertex {
    float4 position [[position]];
    float4 color;
    ...
)");
```

Reply:
oh that is much nicer, I'll do that, thank you

Reply:
The linter did not like that:
```
ERROR: Source/WebGPU/WebGPU/ShaderModule.mm:86:  Multi-line string ("...") found.  This lint script doesn't do well with such strings, and may give bogus warnings.  They're ugly and unnecessary, and you should use concatenation instead".  [readability/multiline_string] [5]
```